packages/components/src/styles/tile-styles.ts

```diff
@@ -0,0 +1,91 @@
+import type { JssStyle, Styles } from 'jss';
+import type { BreakpointCustomizable } from '../types';
+import { buildResponsiveStyles } from '../utils';
+import {
+  addImportantToEachRule,
+  getBackfaceVisibilityJssStyle,
+  getInsetJssStyle,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+} from './';
+import { borderRadiusMedium, spacingStaticMedium } from '@porsche-design-system/utilities-v2';
+import type { TileAspectRatio } from '../utils';
+
+const aspectRatioPaddingTop: Record<TileAspectRatio, string> = {
+  '1:1': '100%',
+  '4:3': '75%',
+  '3:4': '133.33%',
+  '16:9': '56.25%',
+  '9:16': '177.75%',
+};
+
+export const getTileStyles = (opts: {
+  aspectRatio: BreakpointCustomizable<TileAspectRatio>;
+  isDisabled?: boolean;
+  additionalHostStyles?: JssStyle;
```

Comment:
Kind of confusing puzzle we got here and especially in the usage of this function. The last 3 parameters have similar names but different types.
Can't we solve this differently? Perhaps 2 out of the 3 tile components have an identical combination of these parameters and we can group and share does beforehand?